### ISSUE TITLE
Ionic 3 Alert Buttons causing dialog stretching in Capacitor.

### ISSUE THREAD

**Issue by IT-MikeS** [ionic-team/capacitor#482]
While running an Ionic 3 app in Capacitor, creating an Alert with buttons will display incorrectly. 
![capture1](https://user-images.githubusercontent.com/20338451/39003698-61b5db20-43c9-11e8-9e46-729c91f80783.PNG)
The workaround I have found is to add the following snippet to the `variables.scss`:
``` css
.alert-button-md .button-inner {
  display: initial !important;
}
```

**Comment by sbannigan** [comment]
Most likely this is due to an issue with Capacitor's implementation of the Android WebView as described in #662

**Comment by mlynch** [comment]
Fixed in 11981f2fb40db40ea4f55d03c2f0904b48d5adb6
